docs/management/ingest-pipelines/ingest-pipelines.asciidoc

```diff
@@ -7,7 +7,7 @@ pipelines that perform common transformations and
 enrichments on your data.  For example, you might remove a field,
 rename an existing field, or set a new field.
 
-You’ll find *Ingest Node Pipelines* in *Management > Elasticsearch*. With this feature, you can:
+You’ll find *Ingest Node Pipelines* in *Stack Management. With this feature, you can:
```

Comment:
Missing closing asterisk after Stack Management. And maybe we should have the whole path?

```
*Stack Management > Ingest*
```